src/quo2/components/avatars/group_avatar.cljs

```diff
@@ -4,25 +4,29 @@
             [react-native.core :as rn]))
 
 (def sizes
-  {:icon {:small 12
-          :medium 16
-          :large 20}
-   :container {:small 20
+  {:icon      {:small  12
+               :medium 16
+               :large  20}
+   :container {:small  20
                :medium 32
-               :large 48}})
+               :large  48}})
 
+;; TODO: this implementation does not support group display picture (can only display default group icon)
```

Comment:
should there be an issue created for this?

Reply:
@J-Son89 right. Created one https://github.com/status-im/status-mobile/issues/14349